sqlmesh/core/context.py

```diff
@@ -1846,15 +1848,77 @@ def create_external_models(self, strict: bool = False) -> None:
                 strict=strict,
             )
 
+    def get_sensitive_fields(self) -> Set[str]:
+        """Returns a set of common sensitive field names"""
+        return {
```

Comment:
There's already a similar list here: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/config/scheduler.py#L110-L125. Can we unify them?

Reply:
Great idea! Thanks for pointing out!